docs/user/openstack/known-issues.md

```diff
@@ -112,3 +112,19 @@ When anti-affinity is a requirement, it is advised to rely on the more stable
 control plane replicas only. Add more nodes as a day-2 operation by assigning
 them a new Server group with "anti-affinity" policy. Use the `ServerGroupID`
 property of the Machine ProviderSpec.
+
+## Requirement to create Control Plane Machines manifests (Kuryr SDN)
+
+Installations with Kuryr SDN can timeout due to changes in the way Kuryr detects
+the OpenStack Subnet used by the cluster's nodes. Kuryr relied on the Network of
+the cluster's nodes Subnet having a specific tag, but the tag was removed causing
+the need to the discover it from the OpenShift Machine objects, which the creation
+is removed on one of the UPI steps. As a workaround, only the control plane machine
+manifests should **not** be removed in the [Remove machines and machinesets][manifests-removal]
```

Comment:
This is hard to read, I suggest you reverse this to point what should be removed, instead of what should not.
```suggestion
is removed on one of the UPI steps. As a workaround, only the compute machine
manifests should be removed in the [Remove machines and machinesets][manifests-removal]
```

Reply:
+1!